scripts/events.py

```diff
@@ -817,138 +827,181 @@ def other_interactions(self, cat):
+
+            # BIOMES                
+            if game.clan.biome == "Mountainous":
+                if cat.status == 'leader':
+                    cause_of_death.extend([
+                        f'{name} lost a life in an avalanche',
+                        f'{name} lost a life in a landslide',
+                        f'{name} was pushed off a cliff and lost a life',
+                        f'{name} accidentally fell off a cliff and lost a life'
+                    ])
                 else:
-                    lostlives = choice([2, 2, 2, 3, 3, 3, 4, 4, 4, 5, 5, 6])
-                    if cat.moons >= 130 and game.clan.current_season in ['Leaf-fall', 'Leaf-bare']:
-                        cause_of_death = [
-                            f'{name} lost {lostlives} lives due to greencough',
-                            f'{name} lost {lostlives} lives due to whitecough',
-                            f'{name} lost {lostlives} lives due to yellowcough',
-                            f'{name} lost {lostlives} lives due to an illness'
-                        ]
-                        game.clan.leader_lives = current_lives - lostlives
-                        cat.die()
-                        game.cur_events_list.append(choice(cause_of_death) + ' at ' + str(cat.moons + 1) + ' moons old.')
+                    cause_of_death.extend([
+                        f'{name} was crushed by an avalanche',
+                        f'{name} was buried under a landslide',
+                        f'{name} fell from a cliff and died'
+                    ])
+            elif game.clan.biome == "Beach":
+                if cat.status == 'leader':
+                    cause_of_death.extend([
+                        f'{name} was washed out to sea and lost a life',
+                        f'{name} was poisoned by a sea creature and lost a life'
+                    ])
+                else:
+                    cause_of_death.extend([
+                        f'{name} was washed out to sea and drowned',
+                        f'{name} was poisoned by a sea creature and died'
+                    ])
+            elif game.clan.biome == "Plains":
+                if cat.status == 'leader':
+                    cause_of_death.extend([
+                        f'{name} fell into a sinkhole and lost a life',
+                        f'{name} fell into a hidden burrow and lost a life',
+                        f'{name} lost a life when a burrow collapsed on them'
+                    ])
+                else:
+                    cause_of_death.extend([
+                        f'{name} fell into a sinkhole and died',
+                        f'{name} fell into a hidden burrow and could not get out',
+                        f'{name} was buried alive when a burrow collapsed on them'
+                    ])
+            elif game.clan.biome == "Forest":
+                if cat.status == 'leader':
+                    cause_of_death.extend([])
```

Comment:
Should probably be at least one cause here

Reply:
Added a possibility